src/NuGet.Services.SearchService/Controllers/SearchController.cs

```diff
@@ -80,11 +81,19 @@ public async Task<HttpResponseMessage> GetStatusAsync(HttpRequestMessage request
                 ShowDebug = debug,
             };
 
-            return await _searchService.V2SearchAsync(request);
+            try
+            {
+                return Json(await _searchService.V2SearchAsync(request));
```

Comment:
Maybe add a helper so we have the try/catch defined once and call the helper in these three places

Reply:
Good suggestion, cleaned up the code quite nicely